zulipterminal/ui_tools/boxes.py

```diff
@@ -107,26 +107,40 @@ def autocomplete_mentions(self, text: str, state: int,
                               prefix_string: str) -> Optional[str]:
         # Handles user mentions (@ mentions and silent mentions)
         # and group mentions.
-        group_typeahead = ['@*{}*'.format(group_name)
-                           for group_name in self.model.user_group_names
-                           if match_groups(group_name, text[1:])]
-
-        users_list = self.view.users
-        user_typeahead = [prefix_string+'**{}**'.format(user['full_name'])
-                          for user in users_list
-                          if match_user(user, text[len(prefix_string):])]
+
+        group_typeahead = []
+        groups_list = []
```

Comment:
I think the names `*_list` aren't as descriptive of what the variables hold. Maybe something like `*_suggestions`  would be more appropriate?
`group_suggestions`, `combined_suggestions`, for instance.

Reply:
We already have a list called `group_typeahead` so `group_suggestions` would be a bit confusing. I've added `group_names` instead

Reply:
I see what you are saying but since the purpose of the list is to hold 'suggestions', `*_suggestions` would be a good alternative. Also, in my opinion, typeahead and suggestions won't be as confusing.  :)

Reply:
Yeah, on second-thought, `_suggestions` sounds better.